modules/gdscript/gdscript_functions.cpp

```diff
@@ -1753,12 +1782,26 @@ MethodInfo GDScriptFunctions::get_info(Function p_func) {
 			mi.flags |= METHOD_FLAG_VARARG;
 			return mi;
 
+		} break;
+		case LOG_ERROR: {
+
+			MethodInfo mi(Variant::NIL, "log_error", PropertyInfo(Variant::STRING, "message"));
```

Comment:
Typo here, should be `"mesege"`.

Reply:
...what?

Reply:
A reference to an in-joke from years past....

Reply:
Oh! I think I know [about it](https://github.com/godotengine/godot/pull/10029).